platform/osx/godot_content_view.h

```diff
@@ -47,8 +47,8 @@
 
 @interface GodotContentView : RootView <NSTextInputClient> {
 	DisplayServer::WindowID window_id;
-	NSTrackingArea *tracking_area;
-	NSMutableAttributedString *marked_text;
+	NSTrackingArea *tracking_area = nullptr;
+	NSMutableAttributedString *marked_text = nullptr;
```

Comment:
Objective-C stuff can't be initialized in the definition. Both of these are already set in the constructor (`- (id)init`), so it's redundant anyway.